File: 2021/fibonacci.pl
# cerner_2tothe5th_2021
# Prints the fibonacci sequence
# Try it out online here - https://tio.run/#perl5
sub fibonacci {
  my $n = shift;
  $n < 3 ? 1 : fibonacci($n-1) + fibonacci($n-2)
}

foreach (1..20) {
  print fibonacci($_), " ";
}

File: 2021/fibonacci.r
# cerner_2tothe5th_2021
# Try it online here - https://tio.run/#r

Fibonacci <- numeric(10)
Fibonacci[1] <- Fibonacci[2] <- 1
for (i in 3:20) Fibonacci[i] <- Fibonacci[i - 2] + Fibonacci[i - 1]
print(Fibonacci)
